### ISSUE TITLE
4.10.3 was released with database files included

### ISSUE THREAD

**Issue by robertklep** [Automattic/mongoose#5303]
4.10.3 in the NPM repo extracts to almost 1GB:

```
$ du -kcs *
925156	package/
```

There are a few, what look to be, temporary directories in `package/tools` that are the culprit:
```
$ du -kcs package/tools/*
307932	package/tools/31000/
307932	package/tools/31001/
307960	package/tools/31002/
```

They seem to contain MongoDB databases.

**Comment by floo51** [comment]
To add some insight on the consequences, apart from disclosing the databases of whoever published the npm package... We found out it broke our automated serverless API deployment because the unzipped package archive was over 250MB (actually about 1GB).

**Comment by kyrylkov** [comment]
We have the same issue.

**Comment by floo51** [comment]
I ended up not using the npm package and changing my `package.json` to:
```
"mongoose": "git@github.com:floo51/mongoose.git",
```

Dirty patch but works for now.

**Comment by ChavaSobreyra** [comment]
Can confirm problem -- this is breaking my AWS lambda builds for being too large

**Comment by vkarpov15** [comment]
Released 4.10.4, please upgrade if this issue is causing you grief :+1:

**Comment by robertklep** [comment]
Thanks, @vkarpov15 :)
